libraries/AP_HAL_ChibiOS/hwdef/common/common_extf_h730.ld

```diff
@@ -74,10 +74,14 @@ SECTIONS
     . = 0;
     _text = .;
 
-    startup : ALIGN(16) SUBALIGN(16)
+    /* Vector table: copied to RAM by startup script */
+    vectors : ALIGN(1024) SUBALIGN(16)
     {
+        __textvectors_base__ = LOADADDR(vectors);
+        __vectors_base__ = .;
         KEEP(*(.vectors))
-    } > default_flash
+        __vectors_end__ = .;
+    } > flashram AT > default_flash
```

Comment:
You need to try building this with the H7RF - it has less RAM and this may not fit.

Wouldn't the vector table be better in DTCM?

Reply:
@andyp1per OK, yes, DTCM should be even better. Let me verify that it works